xdsl/dialects/irdl/irdl_to_pyrdl.py

```diff
@@ -33,11 +33,15 @@ class {op.sym_name.data}(ParametrizedAttribute{type_addition}):
     return res
 
 
+def get_op_name(op: OperationOp) -> str:
```

Comment:
```suggestion
def class_name_from_op_name(op: OperationOp) -> str:
```

Reply:
I would recommend adding a separate unit test for this, and to handle the case `dialect.part1.part2_bla`

Reply:
I want to move it somewhere more generic to reuse in the next PR, is `dialects/irdl/utils.py` a decent place?

Reply:
Can a symbol name have a `.` in it?

Reply:
Seems so